extensions/rich_text_components/Link/directives/link_directive.html

```diff
@@ -1,7 +1,8 @@
 <!--
   The extra comments in this file are needed to prevent extra whitespace from
   being added before and after the link. See http://stackoverflow.com/q/5078239
--->
-<a ng-if="showUrlInTooltip" href="<[url]>" target="_blank" uib-tooltip="<[url]>" tabindex="<[tabIndexVal]>"><[text]></a><!--
+--><a ng-if="showUrlInTooltip" href="<[url]>" target="_blank" uib-tooltip="<[url]>" tabindex="<[tabIndexVal]>"><[text]></a><!--
+
```

Comment:
Why the extra lines here and on line 7?

Reply:
@seanlip There was some confusion here. I had pushed another commit which fixes the error but somehow that got skipped. I have fixed this now.